app/views/teachers/TeacherStudentView.coffee

```diff
@@ -406,6 +406,7 @@ module.exports = class TeacherStudentView extends RootView
     date = if expires? then moment(expires).utc().format('l') else ''
     utils.formatStudentLicenseStatusDate(status, date)
 
+  canViewStudentProfile: () -> !@classroom || (@classroom.get('ownerID') != me.id && !me.isAdmin())
```

Comment:
The name of the method looks like it will return true if the user can view student profile, but if we are doing the opposite here, we can rename the method to depict that. Or, can change the condition accordingly.

Reply:
🤦‍♂️ Fixed 😄